src/pykeen/regularizers.py

```diff
@@ -37,6 +38,9 @@ class Regularizer(nn.Module, ABC):
     #: Should the regularization only be applied once? This was used for ConvKB and defaults to False.
     apply_only_once: bool
 
+    #: Has this regularizer been updated since last being reset?
+    updated: bool
```

Comment:
one question: what is this for?

Reply:
well we did it for all of the other variables! it's for better documentation

unless you were asking what the variable itself does- I think you implemented this ;)